data/base-filters.yaml

```diff
@@ -424,3 +424,14 @@ filter_strings:
 - '^Commenting favorably on the skills you already possess. \w+ gives you some tips on .*$'
 - '^The fire shard glances off .* shield\.$'
 - '^You begin chanting a psalm to invoke the .* spell\.$'
+- '^Holding mana in this room is becoming very difficult\.  Perhaps you should move on\?$'
+- '^No miners appear to be nearby, and you will be unable to deed resources mined in this area\.$'
+- '^An attendant arrives, takes your deed and motions to two burly workers\.'
+- '^The mental strain of initiating a cyclic spell so recently prevents you from formulating the spell pattern\.$'
+- '^A few small orbiting slivers of lunar magic'
+- '^Wouldn.t it be better if you used a melee weapon\?$'
+- '^A .+ collapses as if dead\.$'
+- '^The fluorescent .+ guides your attack\.$'
+- '^Something in the area interferes with your spell preparations\.$'
+- '^Trabe Chalice'
```

Comment:
💡 This is a bit brief, is there a more meaningful message to match?

Reply:
The full message is something like `Trabe Chalice (Shattered, 28 minutes)`

Reply:
This is fine then